SPECS/opensm/opensm.spec

```diff
@@ -1,34 +1,39 @@
 %global __remake_config 1
-
-Name:    opensm
-Version: 3.3.23
-Release: 2%{?dist}
-Summary: OpenIB InfiniBand Subnet Manager and management utilities
-License: GPLv2 or BSD
+Summary:        OpenIB InfiniBand Subnet Manager and management utilities
+Name:           opensm
+Version:        3.3.23
+Release:        3%{?dist}
+License:        GPLv2 OR BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL:     https://github.com/linux-rdma/opensm
-
-Source0: https://github.com/linux-rdma/opensm/releases/download/%{version}/%{name}-%{version}.tar.gz
-Source2: opensm.logrotate
-Source4: opensm.sysconfig
-Source5: opensm.service
-Source6: opensm.launch
-Source7: opensm.rwtab
-Source8: opensm.partitions
-
+URL:            https://github.com/linux-rdma/opensm
+Source0:        https://github.com/linux-rdma/opensm/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source2:        opensm.logrotate
+Source4:        opensm.sysconfig
+Source5:        opensm.service
+Source6:        opensm.launch
+Source7:        opensm.rwtab
+Source8:        opensm.partitions
+BuildRequires:  bison
+BuildRequires:  byacc
+BuildRequires:  flex
 BuildRequires:  gcc
-BuildRequires: libibumad-devel, systemd, systemd-units
-BuildRequires: bison, flex, byacc
-%if %{__remake_config}
-BuildRequires: libtool, autoconf, automake
-%endif
-Requires: %{name}-libs%{?_isa} = %{version}-%{release}, logrotate, rdma
+BuildRequires:  libibumad-devel
+BuildRequires:  systemd
+BuildRequires:  systemd-units
+Requires:       %{name}-libs%{?_isa} = %{version}-%{release}
+Requires:       logrotate
+Requires:       rdma
 Requires(post): systemd
-Requires(preun): systemd
 Requires(postun): systemd
+Requires(preun): systemd
 # RDMA is not currently built on 32-bit ARM: #1484155
-ExcludeArch: s390 %{arm}
+ExcludeArch:    s390 %{arm}
```

Comment:
We're not going to be building for these architectures, we might as well clean up and remove these lines.

Reply:
removed.